docs/development/sync-protocol.md

```diff
@@ -273,7 +275,8 @@ sequenceDiagram
       "markowsky@ttys001",
       "markowsky@ttys003"
     ],
-    "team_id": "XXXXXXXXXX"
+    "team_id": "XXXXXXXXXX",
+    "signing_id": "XXXXXXXXXX:YYY"
```

Comment:
We should use the Google Team ID and `com.google.Chrome` to make this a more concrete example and match the other docs.

Reply:
Good point, team_id was already like that so I wasn't sure about messing with it.  I'll change now.